### PR TITLE
chore: adding example how to use opentracing with opentelemetry and export to ingest

### PR DESCRIPTION
**Pull request by obecny**

### CODE REVIEW

File: js-opentracing/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const { CollectorTraceExporter } = require('@opentelemetry/exporter-collector');
+const tracerProvider = new NodeTracerProvider();
+const axios = require('axios');
+
+// TraceShim is needed to join OpenTelemetry with OpenTracing
+const { TracerShim } = require('@opentelemetry/shim-opentracing');
+const opentracing = require('opentracing');
+
+tracerProvider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+tracerProvider.addSpanProcessor(new SimpleSpanProcessor(new CollectorTraceExporter({
```
Comment (kayousterhout) [inline js-opentracing/README.md:18]: is it possible to do this with the launcher?

Comment (obecny) [reply js-opentracing/README.md]: done

File: js-opentracing/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+} = require('lightstep-opentelemetry-launcher-node');
+const opentracing = require('opentracing');
+const axios = require('axios');
+
+// TraceShim is needed to join OpenTelemetry with OpenTracing
+const { TracerShim } = require('@opentelemetry/shim-opentracing');
+
+const sdk = lightstep.configureOpenTelemetry({
+  accessToken: 'YOUR_TOKEN',
+  serviceName: 'CollectorTraceExporter with OpenTracing',
```
Comment (kayousterhout) [inline js-opentracing/index.js:13]: can you call this js-otshim-example' for consistency with how the other services in these examples are named?

**Review by kayousterhout — CHANGES_REQUESTED** [review]
We're trying to have consistency across the examples in the different languages / setups; to that end, can you:
(1) move this to a `node/ot-shim` directory?
(2) make this have a client / server, consistent with the other examples, that get spun up by the docker compose that runs everything here?

File: nodejs/ot-shim/client/client.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  lightstep,
+  opentelemetry,
+} = require('lightstep-opentelemetry-launcher-node');
+const opentracing = require('opentracing');
+const { TracerShim } = require('@opentelemetry/shim-opentracing');
+
+const TARGET_URL = process.env.TARGET_URL || 'http://localhost:8080/ping';
+const sdk = lightstep.configureOpenTelemetry();
+
+// development purposes
```
Comment (kayousterhout) [inline nodejs/ot-shim/client/client.js:13]: did you intend to leave this in?

File: nodejs/ot-shim/client/client.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+  const otelTracer = opentelemetry.trace.getTracer('nodejs-ot-shim');
+  // Init OpenTracing Global Tracer with OpenTelemetry Tracer
+  opentracing.initGlobalTracer(new TracerShim(otelTracer));
+  // Now get the Open Tracing global tracer
+  const tracer = opentracing.globalTracer();
+
+  setInterval(() => {
+    const span = tracer.startSpan('client.ping');
+    // const span = otelTracer.startSpan('client.ping');
```
Comment (kayousterhout) [inline nodejs/ot-shim/client/client.js:30]: rm?

**Review by kayousterhout — APPROVED** [review]

File: nodejs/ot-shim/client/client.js (hunk 3 above)
Comment (obecny) [reply nodejs/ot-shim/client/client.js:13]: yes in development this was easier - you can't debug the docker

**Review by mwear — APPROVED** [review]

**Pull request merged**